olp-cpp-sdk-core/include/olp/core/client/RetrySettings.h

```diff
@@ -67,9 +67,32 @@ struct CORE_API RetrySettings {
   int max_attempts = 3;
 
   /**
-   * @brief The connection timeout limit (in seconds).
+   * @brief Maximum time for request to complete (in seconds).
+   *
+   * The default value is 90.
+   *
+   * @note Connection or data transfer will be interrupted after specified
+   * period of time ignoring `connection_timeout` and `transfer_timeout` values.
+   */
+  int timeout = 90;
+
+  /**
+   * @brief Time allowed to wait for connection to establish (in milliseconds).
+   *
+   * The default value is 60 seconds.
+   *
+   * @note The value should be smaller than `timeout`.
```

Comment:
And what if it is larger? It is more of a suggestion I guess

Reply:
The consequences are described in timeout field description.